src/key_file.rs

```diff
@@ -87,4 +88,16 @@ impl KeyFile {
         }
     }
 
+    pub fn get_boolean_list(&self, group_name: &str, key: &str) -> Result<Vec<bool>, Error> {
+        unsafe {
+            let mut length = mem::uninitialized();
+            let mut error = ptr::null_mut();
+            let ret = ffi::g_key_file_get_boolean_list(self.to_glib_none().0, group_name.to_glib_none().0, key.to_glib_none().0, &mut length, &mut error);
+            if !error.is_null() {
+                return Err(from_glib_full(error));
+            }
+            let v: Vec<ffi::gboolean> = FromGlibContainer::from_glib_container_num(ret, length as usize);
```

Comment:
I thought we had a `FromGlibContainer` implementation for `bool` already

Reply:
No, there wasn't one.  There was a `FromGlib<glib_ffi::gboolean>` for `bool`.  But there was nothing to translate a `*mut gboolean` to a `Vec<bool>`.  I added one in the next commit.